common/types/explorer.ts

```diff
@@ -401,3 +401,8 @@ export type MOMENT_UNIT_OF_TIME =
   | 's'
   | 'milliseconds'
   | 'ms';
+
+export interface GridSortingColumn {
+  id: string;
+  direction: 'asc' | 'desc';
+}
```

Comment:
this is good but prefer to reuse existing type declaration `EuiDataGridSorting['columns']`, so we don't need to change type if upstream changes

Reply:
modified